cmd/deploy.go

```diff
@@ -50,6 +50,28 @@ type Spec struct {
 	ApplicationImage string `yaml:"applicationImage"`
 }
 
+func findNamespaceRepositoryAndTag(image string) string {
+	var nameSpaceRepositoryAndTag string
+	if strings.Count(image, "/") > 1 {
```

Comment:
Could this have been done with `strings.Split(image, "/")`?

Reply:
yes please re-review